src/python/pants/option/global_options.py

```diff
@@ -273,6 +273,16 @@ def register_bootstrap_options(cls, register):
       help='Create a new pantsd server, and use it, and shut it down immediately after. '
            'If pantsd is already running, it will shut it down and spawn a new instance (Beta)')
 
+    # This determines how long a request to the Pailgun Server could block for before failing.
```

Comment:
I don't think this comment is necessary, because the code below / official `help` message describe this already.